HotelReservation/Api/Startup.cs

```diff
@@ -27,14 +28,37 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+            
             services.AddDbContext<Context>(opt =>
-                opt.UseInMemoryDatabase("users"));
+                opt.UseSqlServer(Configuration.GetConnectionString("HotelContextConnection"),
+                    x => x.MigrationsAssembly("Api")));
+            services.AddScoped<HashPassword>();
+            services.AddScoped<ITokenService, TokenService>();
+            services.AddScoped<IAuthenticationService, AuthenticationService>();
             services.AddControllers();
+            services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
+                .AddJwtBearer(options =>
+            {
+                options.RequireHttpsMetadata = false;
+                options.TokenValidationParameters = new TokenValidationParameters
```

Comment:
extract TokenValidationParameters to appsettings

Reply:
extracted